### DISCUSSION TITLE
TAC et certification Google CMP

### DISCUSSION THREAD

**Discussion by selim-bakir** [AmauriC/tarteaucitron.js#1103] [General]
Bonjour @AmauriC ,

Est ce que vous avez fait la demande de certification google qui à était mise en place le 17mai et qui va etre obligatoire pour les outils CMP à avoir pour continuer à utiliser les outils comme : Google AdSense, Ad Manager ou AdMob  ?

[lien de l'article ici](https://support.google.com/adsense/thread/216186168/nouvelles-exigences-pour-diffuser-des-annonces-dans-l-eee-et-le-royaume-uni?hl=fr)

[Lien du formulaire pour soumission de la demande de certification](https://docs.google.com/forms/d/17dUlPV04IPreRnv6um4jCZzxhxBvzSjL_QxuIFYrQBs/viewform?edit_requested=true)

Merci

**Comment by AmauriC** [comment]
Bonjour,

Je suis bien au courant de la nouveauté de Google mais non, je ne ferai pas de demande pour être certifié.

Je ne suis certifié que par la communauté des utilisateurs, ça permet de faire évoluer tarteaucitron selon les vrais besoins et pas ceux des adservers qui veulent optimiser. C'est un point qui me tiens à cœur 😚

Plus concrètement, Google va imposer l'utilisation d'un CMP conforme TCF, ce que j'ai toujours refusé d'être. C'est le type de CMP qui demande le consentement pour des dizaines de sites tiers sans pouvoir facilement refuser, tout ce qui est "Gray pattern" et que Google adore. 

Charger Adsense ou les autres outils publicitaires de Google ne sera bientôt plus autorisé par Google via tarteaucitron.
Je n'ai aucune idée de comment cette règle sera appliquée 😲

A suivre..

**Reply by selim-bakir** [reply]
c'est noté :) je comprend totalement.

**Comment by sanzefor** [comment]
Boujour les gentils !

pour info, depuis ce matin, l'accès à l'espace adsense est bloqué tant qu'on ne répond pas à ce QCM.
Qui a dit que les QCM permettaient un apprentissage ludique ? 😲

![adsense](https://github.com/AmauriC/tarteaucitron.js/assets/102414164/9a139a66-ba47-4dd2-b902-a37e68a71570)

**Comment by AmauriC** [comment]
> Si une PGC réussit l'examen par rapport aux critères de certification Google, elle sera disponible dans notre liste des PGC certifiées par Google. **Google ne vérifie pas la conformité totale des PGC avec** le TCF ou **le droit relatif à la confidentialité**.

Quitte à faire un examen et forcer l'usage, c'est dommage 🥺

tarteaucitron n'utilisera toujours pas le TCF donc ne pourra pas être dans cette liste..

**Reply by sanzefor** [reply]
Si je choisis de cliquer sur le bouton central, parce que je suis fainéant et vénal...
...et que je garde l'annonce de TaC, parce que j'aime les gentils... 
...il y aura **deux** cartons et donc deux clics pour les visiteurs? 

Ça va faire lourd au niveau esthétique. La page d'**accueil** restera une page, mais bon, pour l'accueil...
Quelle aventure ! 😂

**Reply by AmauriC** [reply]
C'est ce que j'ai testé et ça affiche bien une seconde bannière :/

**Reply by sanzefor** [reply]
Merci 👍

**Comment by chaissac** [comment]
C'est triste cette situation, avec en plus l'échéance de l'abandon des cookies tiers en mars prochain, toutes les boites comme la mienne qui font des campagnes digitales via Google vont être obligées d'abandonner tarteaucitron et aller vers une solution certifiée 😭
Il n'y pas d'alternative ?
